README.md

```diff
@@ -17,6 +17,10 @@ This project is part of ASP.NET Core. You can find samples, documentation and ge
 
 To run a complete build on command line only, execute `build.cmd` or `build.sh` without arguments. See [developer documentation](https://github.com/aspnet/Home/wiki) for more details.
 
+## File logging for functional test
+
+Turn on file logging for Kestrel functional tests by specifying the environment variable ASPNETCORE_TEST_LOG_DIR to the log output directory.
```

Comment:
Can we default to dropping logs in `{TestProjectDirectory}/testlogs` or something like that? We could still have an environment variable to opt-out if it's really needed.

Reply:
Hmm I need to think about where this default setting should live. Might need to make this change in Logging.

Reply:
There's a couple of ways to do this so let's talk about what we really want here. But we don't need to necessarily wait to figure this out for this PR and we can do this separately afterwards.